setup.py

```diff
@@ -360,7 +360,7 @@ def _download(self, pkg):
         from taucmdr import util
         module_name = pkg + '_installation'
         repos = getattr(__import__('.'.join(('taucmdr', 'cf', 'software', module_name)),
```

Comment:
Not important now, but the documentation for `__import__` says that "Direct use of `__import__` is also discouraged in favor of `importlib.import_module()`". Does this have to be `__import__` or could it be replaced by something not discouraged by the Python documentation? https://docs.python.org/3/library/functions.html#import__

Reply:
yes, actually there is a certain amount of this, and I recall that doing a direct replacement didn't work, but it's on the todo list of things to look into and resolve. I'll run some experiments now if my memory is correct and it's not straight forwards, then I'll merge this PR and open a new issue